src/helpers/searchHelper.ts

```diff
@@ -0,0 +1,29 @@
+export type SearchParams = {
+  [key: string]: string | string[] | null,
+};
+
+export function getSearchWith(
```

Comment:
You can name the file and your helper named (for example `getSearchWith.ts`). 
So you can easier manage your project structure.